tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/FSharp.Compiler.Benchmarks.fsproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
+    <ExcludeFromDotNetBuild Condition="'$(DotNetBuildTests)' != 'true'">true</ExcludeFromDotNetBuild>Z
```

Comment:
Remove the Z at the end. Note that fsharp is on Arcade 8 so you might need to use a different property here.

Reply:
Arcade 8 isn't used in context of building this in the VMR (and this is excluded elsewhere in SB).